webroot/src/network/userApi/data.api.ts

```diff
@@ -174,6 +174,25 @@ export class UserDataApi implements DataApiInterface {
         return response;
     }
 
+    /**
+     * UPDATE Password of authenticated user.
+     * @param  {object}          data The request data.
+     * @return {Promise<object>}      Axios-formatted response from AWS Cognito.
+     */
+    public async updateAuthenticatedUserPassword(data) {
```

Comment:
nit but can we explicitly type this and L211 as `any` just so it's clear that that was the intent? (If this is a longer-term practice that folks agree to, perhaps we can enable the `no-implicit-any` lint rule.)

Reply:
Added